requirements/systemmodel/system-model.tex

```diff
@@ -0,0 +1,24 @@
+\subsection{System Model}
+\begin{figure}
+  \subsubsection{Schedule Task}
+  \centering
+  \includegraphics[width=\linewidth]{./systemmodel/scheduletask.png}
+  \caption{This diagram depicts how a user schedules a task, which the CS distributes to WCM 1.
+  In that case CS knows that there are enough resurces on WCM 1 to execute users task.
+  At the end the CS notifies the user that the task has ended.}
+\end{figure}
+
+\begin{figure}
+  \subsubsection{Request access to task}
+  \centering
+  \includegraphics[width=\linewidth]{./systemmodel/getaccess.png}
+  \caption{This diagram depicts how a user accesses his running task.
+  The user has to be the user that started the task.}
```

Comment:
Or can we just do  `Server returns work machine IP address and container ID.` when the job starts, so that the user can access it whenever he wants.

about 

> Also, what about privileged users/administrators?
Do you mean that there should be a diagram how the administrators access jobs?

Reply:
Regarding adminstrators:
You wrote `The user has to be the user that started the task.`
I think an administrator should be allowed to access all running tasks.